pkg/state/init.sql

```diff
@@ -89,13 +89,13 @@ BEGIN
                     'oid', t.oid,
                     'comment', descr.description,
                     'columns', (SELECT COALESCE(json_object_agg(name, c), '{}'::json)
-                                FROM (SELECT attr.attname                                                                                        AS name,
```

Comment:
I'm not sure what's going on with the formatting. Running `make format` doesn't seem to do anything